amp/WEB-INF/src/org/digijava/module/aim/dbentity/AmpActivityFields.java

```diff
@@ -146,7 +153,7 @@
     @VersionableFieldSimple(fieldTitle = "Document Space")
     protected String documentSpace;
 
-    @Interchangeable(fieldTitle = ActivityFieldsConstants.IS_DRAFT, required="_ALWAYS_", importable=false)
+    @Interchangeable(fieldTitle = ActivityFieldsConstants.IS_DRAFT)
```

Comment:
Again, no longer required?

Reply:
required is not used when importable is false

Reply:
I recall that we have to specify draft when importing from AMP Offline. I found however that there is special case handled through OFFLINE_REQUIRED_FIELDS (that btw also includes approval status).